File: swagger/models/issue_response.py
from .base_model import BaseModel

from .user import User

class IssueResponse(BaseModel):
	"""
	POST UserToken/Issue successful response

	Attributes:

		access_token
		The authentication token value.

		user
		Contains information about the user represented by the access token.
	"""

	input_map = {
		'TokenType': 'token_type',
		'AccessToken': 'access_token',
		'User': ('user', User)
		}

	output_map = {
		'token_type': 'TokenType',
		'access_token': 'AccessToken',
		'user': 'User',
		}

File: swagger/Models/IssueResponse.php
<?php

namespace App\MindBody\Models;

/**
 * POST UserToken/Issue successful response
 */
class IssueResponse extends BaseModel
{
	/**
	 * no description available
	 * 
	 * @var string
	 */
	public $tokenType;
	/**
	 * The authentication token value.
	 * 
	 * @var string
	 */
	public $accessToken;
	/**
	 * Contains information about the user represented by the access token.
	 * 
	 * @var User
	 */
	public $user;

	protected function getInputMap()
	{
		return [
			'TokenType' => 'tokenType',
			'AccessToken' => 'accessToken',
			'User' => ['user', User::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'tokenType' => 'TokenType',
			'accessToken' => 'AccessToken',
			'user' => 'User',
			];
	}
}
